app/views/user_mailer/in_person_please_call.html.erb

```diff
@@ -8,7 +8,7 @@
 <p>
   <%= t(
         'user_mailer.in_person_please_call.body.intro_html',
-        date: I18n.l(14.days.from_now, format: I18n.t('time.formats.event_date')),
+        date: I18n.l(14.days.from_now, format: I18n.t('time.formats.full_date')),
```

Comment:
love how this is simpler and consistent across all locales! thank you

Reply:
agreed, thank you for this suggestion!